samples/project-with-buildSrc/buildSrc/build.gradle.kts

```diff
@@ -1,7 +1,16 @@
 plugins {
     `kotlin-dsl`
+
+    // see buildSrc/src/main/kotlin/allopen-compiler-plugin.kt
+    id("org.jetbrains.kotlin.plugin.allopen") version "1.3.0"
```

Comment:
```suggestion
    id("org.jetbrains.kotlin.plugin.allopen") version KotlinVersion.CURRENT.toString()
```

I know this is different from what the other plugins use, but it will keep the plugin version in sync with the version of Kotlin that the Kotlin DSL is using.

Reply:
Ok! Should I use the `kotlin("allopen")` syntax?

Reply:
Not sure. Sorry.

Reply:
Updated to `kotlin("plugin.allopen") version KotlinVersion.CURRENT.toString()`